1001-2000/LIT1233Cantic.xml

```diff
@@ -33,6 +33,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract><p>The English title Canticles refers to two texts, 
+            <ref type="work" corresp="LIT1828Mahale"/> and <ref type="work" corresp="LIT2362Songof"/> 
```

Comment:
I wonder if, in keeping with Sophia's consistency proposal, we make Book of Odes (https://en.wikipedia.org/wiki/Book_of_Odes_(Bible)) the main title for LIT1828